satchless/payment/__init__.py

```diff
@@ -4,6 +4,14 @@ def __init__(self, error_message):
         self.error_message = error_message
 
 
+class PaymentsGatewayReceiptFormError(PaymentFailure):
+    def __init__(self, form):
+        error_message = ', '.join(
```

Comment:
Think you may want u'' here for the separator, errors tend to be unicode.

Could also switch from `items` to `iteritems` for efficiency.